csrc/ec_utils.cpp

```diff
@@ -97,16 +97,6 @@ JNIEXPORT jint JNICALL Java_com_amazon_corretto_crypto_provider_EcUtils_curveNam
         BigNumObj orderBN;
 
         const EC_POINT* generator = NULL;
```

Comment:
All the changes to this file don't seem relevant to SecureRandom's default value. What's going on here?

Reply:
[EC_METHOD_get_field_type always returns NID_X9_62_prime_field](https://github.com/aws/aws-lc/blob/AWS-LC-FIPS-2.0.13/include/openssl/ec.h#L369). In AWS-LC v1.30.1 it is marked as deprecated, so I've removed it and simplified the code according to the fact that it always returns `NID_X9_62_prime_field`.

Reply:
For the future, it would help to explain this kind of stuff in the PR description so there's less confusion. It would also be good hygiene to have separate PRs for the secureRandom change and bringing LC up to date.

Reply:
I totally agree. I think we should update LC versions more often in separate PRs.